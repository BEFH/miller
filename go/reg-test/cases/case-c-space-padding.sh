# ----------------------------------------------------------------
announce SPACE-PADDING

run_mlr --idkvp    --odkvp --ifs space --repifs cat $indir/space-pad.dkvp
run_mlr --inidx    --odkvp --ifs space --repifs cat $indir/space-pad.nidx
run_mlr --icsvlite --odkvp --ifs space --repifs cat $indir/space-pad.pprint
