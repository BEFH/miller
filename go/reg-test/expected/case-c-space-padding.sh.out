

================================================================
SPACE-PADDING

mlr --idkvp --odkvp --ifs space --repifs cat ././reg-test/input/space-pad.dkvp
a=1,b=2
c=3

mlr --inidx --odkvp --ifs space --repifs cat ././reg-test/input/space-pad.nidx
1=a,2=b,3=c
1=d,2=e,3=f

mlr --icsvlite --odkvp --ifs space --repifs cat ././reg-test/input/space-pad.pprint
a=1,b=2,c=3

